commands/inspect_test.go

```diff
@@ -29,7 +29,7 @@ func TestCmdInspectFormat(t *testing.T) {
 	assert.Equal(t, "\"none\"", actual)
 
 	actual, _ = runInspectCommand(t, []string{"--format", "{{prettyjson .Driver}}"})
```

Comment:
Any reason to use `prettyjson` instead of the usual format here?  Might save you some whitespace headaches